src/pages/docs/deployments/terraform/working-with-built-in-steps/index.md

```diff
@@ -77,6 +80,30 @@ Given that Terraform templates and variable files are plain text, you may find i
 ![Package](/docs/deployments/terraform/working-with-built-in-steps/images/step-aws-package.png)
 :::
 
+### Git repository
+
+:::div{.info}
+Octopus version `2024.1` added support for terraform files stored in Git repositories.
```

Comment:
A couple lowercase terraforms in here. I'm not sure what our stance is on capitlisation there. Not my domain.

Reply:
Hmmm I'll check out what the rest of the docs do and match that, thanks

Reply:
Looks like its capitalised